src/components/common/AddressLink/styles.module.scss

```diff
@@ -12,7 +12,7 @@
     display: flex;
     justify-content: flex-start;
     align-items: center;
-    min-width: 8vw;
+    margin-left: 8vw;
```

Comment:
there's no space now between these 2 elements , instead of `margin-left` give `margin-right` as i said previously it should work perfectly ✨ 

<img width="230" alt="Screenshot 2022-02-16 at 2 17 55 AM" src="https://user-images.githubusercontent.com/55599878/154146689-59cd3f30-6d29-4187-9c7e-29f965a4c968.png">

Reply:
Should be done right this time, thank you for taking the time.